src/TicketSwapErrorFormatter.php

```diff
@@ -180,13 +180,15 @@ public static function link(
 
     private static function trimPath(string $path) : string
     {
-        $parts = explode(DIRECTORY_SEPARATOR, $path);
+        // Normalize to forward slashes for consistent handling across platforms
+        $normalizedPath = str_replace('\\', '/', $path);
+        $parts = explode('/', $normalizedPath);
         if (count($parts) < 6) {
             return $path;
         }
 
         return implode(
-            DIRECTORY_SEPARATOR,
+            '/',
```

Comment:
Doesn't this produce weird trimmed paths on windows with forward slashes?

Reply:
But I think seeing `src/File.php` on windows will look weird. People on windows expect to see `src\File.php` right?

Reply:
before this change, it already shown as `src/File.php`, seems happen on internal phpstan, it just to fix unit test, do you think unit test needs to be updated instead?

Reply:
It's because I never think of windows. I think it makes sense that if we test it on windows the output also looks windows native. So yes then the test should be altered to handle both OS.

Reply:
I've rolled back to original use of DIRECTORY_SEPARATOR on commit https://github.com/TicketSwap/phpstan-error-formatter/pull/30/changes/b7875a9edfb356e99b08b1192fd817c3581b9f95

I've then updated unit test to cover under windows os https://github.com/TicketSwap/phpstan-error-formatter/pull/30/changes/cd151051eb1399384f96bc31d733d0c09fbeb37b